src/sentry/services/hybrid_cloud/app.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+import abc
+from dataclasses import dataclass, field
+from typing import List
+
+from sentry.constants import SentryAppInstallationStatus
+from sentry.models import SentryApp, SentryAppInstallation
+from sentry.services.hybrid_cloud import (
+    CreateStubFromBase,
+    InterfaceWithLifecycle,
+    silo_mode_delegation,
+)
+from sentry.silo import SiloMode
+
+
+@dataclass
+class ApiSentryApp:
+    id: int = -1
+    scope_list: List[str] = field(default_factory=list)
+    application_id: int = -1
+    proxy_user_id: int | None = None  # can be null on deletion.
+    owner_id: int = -1  # relation to an organization
+    name: str = ""
+    slug: str = ""
+    uuid: str = ""
```

Comment:
We might need the event list as it gets used in regions to create service hooks.

Reply:
okie doke

Reply:
Added